### PR TITLE
Unit testing

### PR DESCRIPTION
**Pull request by chidghanjadhav-ugam**

### CODE REVIEW

File: core/src/test/java/com/ugam/core/models/impl/BannerAreaImplTest.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+import static org.junit.jupiter.api.Assertions.*;
+
+@ExtendWith({AemContextExtension.class, MockitoExtension.class})
+class BannerAreaImplTest {
+    private final AemContext aemContext = new AemContext();
+    private BannerArea banner;
+
+    @BeforeEach
+    void setUp() {
+        //aemContext.load().json(BannerAreaImpl.class.getResourceAsStream ("/com/ugam/core/models/impl/Banner.json"),"/component");
```
Comment (vamshikega) [inline core/src/test/java/com/ugam/core/models/impl/BannerAreaImplTest.java]: remove comments

File: core/src/test/java/com/ugam/core/models/impl/BannerAreaImplTest.java [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    @Test
+    void getTitle() {
+        Resource json = aemContext.currentResource("/content/banner");
+        BannerArea banner = json.adaptTo(BannerArea.class);
+        assertEquals("THIS IS ME", banner.getTitle());
+    }
+
+    @Test
+    void getName() {
+//        Resource json = aemContext.resourceResolver().getResource("/content");
```
Comment (vamshikega) [inline core/src/test/java/com/ugam/core/models/impl/BannerAreaImplTest.java]: remove comments

File: core/src/test/java/com/ugam/core/models/impl/BannerAreaImplTest.java (hunk 1 above)
Comment (chidghanjadhav-ugam) [reply core/src/test/java/com/ugam/core/models/impl/BannerAreaImplTest.java]: i have removed the comments

**Review by vamshikega — APPROVED** [review]
approved

**Pull request merged**